Sources/Authentication/Header/Basic.swift

```diff
@@ -1,21 +1,25 @@
-import Core
-
 extension AuthorizationHeader {
     public var basic: Password? {
         guard let range = string.range(of: "Basic ") else {
             return nil
         }
-
+        
```

Comment:
fix spacing. make sure to have this enabled in xcode :)

![screen shot 2017-07-19 at 7 11 41 pm](https://user-images.githubusercontent.com/1342803/28393678-1fea5080-6cb6-11e7-9d10-642459023073.png)